Sources/StringExtensions.swift

```diff
@@ -452,7 +452,11 @@ extension String {
     }
     
     #endif
-
+    
+    // EZSE: URL encode a string (percent encoding special chars)
+    public func urlEncode() -> String {
```

Comment:
Should be urlEncoded, urlEncode otherwise mutate the string.
Refer ro array.sort(ed)